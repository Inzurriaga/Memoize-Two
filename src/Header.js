import React, { Component} from "react"

export default class Header extends Component{
    togglePage = e => {
        this.props.togglePage(e.target.value)
    }

    signOut = () => {
        this.props.signOut()
    }

    render(){
        if(this.props.user !== undefined){
        return(
            <div className="header-section">
                    <h1>Javascript War</h1>
                    <nav>
                        <input className="page-button" onClick={this.togglePage} type="button" value="home"></input>
                        <input className="page-button" onClick={this.togglePage} type="button" value="train"></input>
                    </nav>
                    <input  onClick={this.signOut} className="signout-button" type="button" value="SIGN OUT"></input>
            </div>
        )
        }else{
            return(
                <div className="header-section">
                    <h1>Javascript War</h1>
                </div>
            ) 
        }
    }
}