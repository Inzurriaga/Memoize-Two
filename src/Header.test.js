import React from 'react';
import { shallow } from 'enzyme';
import Header from "./Header"

describe("Header", ()=> {
  it("should pass test", () => {
    expect(true).toEqual(true)
  })
})