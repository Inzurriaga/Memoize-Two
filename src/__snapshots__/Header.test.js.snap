// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should hide buttons if no player is sign in 1`] = `
<div
  className="header-section"
>
  <h1>
    Javascript War
  </h1>
</div>
`;

exports[`Header should match Snap shot 1`] = `
<div
  className="header-section"
>
  <h1>
    Javascript War
  </h1>
  <nav>
    <input
      className="page-button"
      onClick={[Function]}
      type="button"
      value="home"
    />
    <input
      className="page-button"
      onClick={[Function]}
      type="button"
      value="train"
    />
  </nav>
  <input
    className="signout-button"
    onClick={[Function]}
    type="button"
    value="SIGN OUT"
  />
</div>
`;
